### PR TITLE
Creates subscription page

### PR DESCRIPTION
**Pull request by angepol**
- creates a subscription page
- componentizes Subscription buttons
- routes sign up button to payment-intent page

### CODE REVIEW

File: cha-ching-app/src/App.js [hunk 1]
```diff
@@ -13,37 +14,40 @@ const stripePromise = loadStripe(process.env.REACT_APP_PUBLIC_STRIPE_KEY);
...
-      </Elements>
-    ) : (
-      showError ? <Error /> : <div>Loading</div>
-    );
+    <Elements options={options} stripe={stripePromise}>
+      <Router>
+        <Routes>
+          <Route path="/" element={<CheckoutForm />} />
+          <Route path="/success" element={<Success />} />
+          <Route path="/subscriptionPage" element={<SubscriptionPage />} />
```
Comment (Ashlynn223) [inline cha-ching-app/src/App.js:42]: we can update this to kabob case

File: cha-ching-app/src/SubscriptionButton.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+.SubscriptionButton {
+  border-radius: 20px;
+  cursor: pointer;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 22px;
+  font-weight: 500;
+  height: 100px;
+  width: 200px;
+}
```
Comment (Ashlynn223) [inline cha-ching-app/src/SubscriptionButton.css:24]: Could you explain why there's two SubscriptionButton classes

File: cha-ching-app/src/SubscriptionPage.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+.title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 32px;
+  margin-top: 12px;
+}
+.buttonsWrapper {
```
Comment (Ashlynn223) [inline cha-ching-app/src/SubscriptionPage.css:16]: We could add a space here

**Review by Ashlynn223 — APPROVED** [review]
Good to merge once the final tweaks have been made :)

File: cha-ching-app/src/SubscriptionButton.css (hunk 2 above)
Comment (angepol) [reply cha-ching-app/src/SubscriptionButton.css:24]: Thanks or pointing this out, this is an accidental duplicate. Initially I had two different classes, one for each button although I missed removing it when I changed the logic. Will remove in the next commit.

**Pull request merged**